plane/source/docs/quadplane-flying.rst

```diff
@@ -203,12 +212,10 @@ What Will Happen?
 Understanding hybrid aircraft can be difficult at first, so below are
 some scenarios and how the ArduPilot code will handle them.
 
-I am hovering in QHOVER and switch to FBWA mode
------------------------------------------------
+I am hovering in QHOVER/QLOITER and switch to FBWA mode
+-------------------------------------------------------
 
-The aircraft will continue to hover, waiting for pilot input. If you
-take your hands off the sticks at zero throttle the aircraft will
-continue to hold the current height and hold itself level. It will drift
+The aircraft will continue to hover, setting forward thrust/throttle at whatever the throttle stick position dictates and gaining speed. If you zero throttle during the transition, the aircraft will continue to hold the current height and hold itself level, slowing to a halt. It will drift
```

Comment:
If it continues to hover with zero throttle stick input, are you still in VTOL mode with the mode switch in FBWA? Does the GCS indicate FBWA? Is there a timeout for this transition? Does it matter if qassist is enabled?

Reply:
no in transition wait
yes
only if set the overall timeout,covered later
yes, qassist is independent and a fixed wing mode phenomenon

Reply:
Seems confusing that zero throttle means hover in this case, although I guess you would normally be at hover throttle when beginning the transition.
I realize that I'm talking about design instead of documentation though, and also showing my ignorance of how things actually work.

Reply:
Yeah.... I welcome Greg's comments...you don't usually see this since throttle is always at hover position when you flip the switch......but I had  snafu with my quadplane that made me want to determine what really goes on..so I sim'd  it. .Rather than always having Tridge be bothered, I just wanted some other plane guy to review